build.cake

```diff
@@ -402,6 +404,90 @@ Task("PackageConsole")
         });
     });
 
+Task("PackageChocolatey")
+	.Description("Creates chocolate packages of the console runner")
+    .IsDependentOn("CreateImage")
+	.Does(() =>
+	{
+		var currentImageDir = IMAGE_DIR + "NUnit-" + packageVersion + "/";
+		
+		EnsureDirectoryExists(PACKAGE_DIR);
+		
+		// Note: Since cake does not yet support a working directory and separate output directory for chocolatey, the following copying and hacks are needed.
+		
+		// List with the addins (addin name, primary dll, additional files)
+		var addins = new Tuple<string, string, string[]>[] {
+			new Tuple<string, string, string[]>(
+				"NUnit.Extension.VSProjectLoader",
+				"vs-project-loader.dll",
+				null
+			),
+			new Tuple<string, string, string[]>(
+				"NUnit.Extension.NUnitProjectLoader",
+				"nunit-project-loader.dll",
+				null
+			),
+			new Tuple<string, string, string[]>(
+				"NUnit.Extension.NUnitV2ResultWriter",
+				"nunit-v2-result-writer.dll",
+				null
+			),
+			new Tuple<string, string, string[]>(
+				"NUnit.Extension.NUnitV2Driver",
+				"nunit.v2.driver.dll",
+				new [] {
+					"nunit.core.dll",
+					"nunit.core.interfaces.dll",
+					"nunit.v2.driver.addins"
+				}
+			),
+			new Tuple<string, string, string[]>(
+				"NUnit.Extension.TeamCityEventListener",
+				"teamcity-event-listener.dll",
+				null
+			)
+		};
+		
+		// Install and copy the addin files
+		var toolsDir = "tools";
+		var nugetInstallSettings = new NuGetInstallSettings { OutputDirectory = toolsDir, ExcludeVersion = true };
+		var addinsDir = System.IO.Path.Combine(currentImageDir, "addins");
+		EnsureDirectoryExists(addinsDir);
+		foreach (var addin in addins) {
+			// Install the extension
+			NuGetInstall(addin.Item1, nugetInstallSettings);
```

Comment:
Let's at least add a comment here, that this installs the latest published version of the extension. It's OK right now, but I think we will want to control the version we install since we don't control the publication of all extensions. We will want to only publish with versions we have tested, at least at some point.